### PR TITLE
0.9 Beta Release

### LINKED ISSUE #488: Create 0.9 Release
n/t

### PR DESCRIPTION
**Pull request by ahankinson**
This is the 0.9 release PR. 

To review: The first commit contains a wholescale copy of the latest draft into the 0.9 folder. Subsequent commits were then applied to change specific parts of this to reflect the updated version number.

Open question: Should we update the validator codes to point to 0.9, or should we leave that for the final 1.0 release?

Fixes #488

### CODE REVIEW

**Comment by zimeon** [comment]
I'd leave the validator codes on `/draft/` until we get to 1.0

File: 0.9/spec/index.html [hunk 1]
```diff
@@ -0,0 +1,1671 @@
...
+            ],
+            edDraftURI: "https://ocfl.io/0.9/spec/",
+            wg: "Oxford Common File Layout Editors",
+            wgURI: "https://ocfl.io",
+            wgPublicList: "",
+            otherLinks: [{
+                key: "Previous version",
+                data: [
+                    {
+                        value: "N/A"
```
Comment (awoods) [inline 0.9/spec/index.html:60]: This `value` should likely point to [https://ocfl.io/0.3/spec/](https://ocfl.io/0.3/spec/).

File: 0.9/implementation-notes/index.html [hunk 2]
```diff
@@ -0,0 +1,923 @@
...
+            ],
+            edDraftURI: "https://ocfl.io/0.9/implementation-notes/",
+            wg: "Oxford Common File Layout Editors",
+            wgURI: "https://ocfl.io",
+            wgPublicList: "",
+            otherLinks: [{
+                key: "Previous version",
+                data: [
+                    {
+                        value: "N/A"
```
Comment (awoods) [inline 0.9/implementation-notes/index.html:60]: This `value` should likely point to [https://ocfl.io/0.3/implementation-notes/](https://ocfl.io/0.3/implementation-notes/)

File: 0.9/spec/index.html (hunk 1 above)
Comment (ahankinson) [reply 0.9/spec/index.html:60]: I thought about that, but wasn't sure if we wanted to do this for beta releases, or only actual releases.

Comment (ahankinson) [reply 0.9/spec/index.html]: It seems that we did that for previous betas, so I've fixed this now.

**Pull request merged**